content/uiux/concepts/crazy-eights/crazy-eights.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Crazy Eights'
+Description: 'A sketching exercise that challenges people to sketch eight distinct ideas in eight minutes'
+CatalogContent:
```

Comment:
Refer [this uiux entry](https://github.com/Codecademy/docs/blob/main/content/uiux/concepts/design-methodologies/terms/five-elements-of-ux-design/five-elements-of-ux-design.md?plain=1) and add `Subjects` and  `Tags` to this entries metadata.

Reply:
done